modules/javafx.web/src/main/native/Source/WTF/wtf/FileSystem.cpp

```diff
@@ -531,8 +531,7 @@ std::optional<Salt> readOrMakeSalt(const String& path)
     return salt;
 }
 
-#if HAVE(STD_FILESYSTEM) || HAVE(STD_EXPERIMENTAL_FILESYSTEM)
-
+#if !PLATFORM(JAVA)
```

Comment:
Can you explain this change? The only way I can see that it would make sense is if all of the following are true:

1. In WebKit620.1 _prior to_ the 2.46.6 changes, neither  `HAVE(STD_FILESYSTEM)` nor `HAVE(STD_EXPERIMENTAL_FILESYSTEM)` were defined in the CMake files.
2. The upstream code removed the conditional compilation
3. We still need to exclude it from our port

Are all three of the above statements true?